DNN Platform/Library/Services/Upgrade/Upgrade.cs

```diff
@@ -3797,7 +3797,7 @@ public static void AddModuleToPages(string tabPath, int moduleDefId, string modu
         /// <remarks>
         /// </remarks>
         /// -----------------------------------------------------------------------------
-        public static int AddPortal(XmlNode node, bool status, int indent)
+        public static int AddPortal(XmlNode node, bool status, int indent, UserInfo superUser = null)

```

Comment:
You are going to need to mark the old AddPortal function obsolete and removed in v11. Otherwise the changes look good! Thanks for tracking down the issue and getting a fix submitted!

Reply:
@ohine  Please review changes.